Makefile

```diff
@@ -0,0 +1,118 @@
+.PHONY: all java java-test python python-test node node-test check-redis-server go go-test
+
+BLUE=\033[34m
+YELLOW=\033[33m
+GREEN=\033[32m
+RESET=\033[0m
+ROOT_DIR=$(shell pwd)
+PYENV_DIR=$(shell pwd)/python/.env
+PY_PATH=$(shell find python/.env -name "site-packages"|xargs readlink -f)
+PY_GLIDE_PATH=$(shell pwd)/python/python/
+
+all: java java-test python python-test node node-test go go-test python-lint java-lint
```

Comment:
node-lint go-lint are missing

Reply:
Already added them in later commit

Reply:
I mean those are are not parts of `all`.
BTW, do you want to add tasks for c# client too?